internal/directives/argocd_revisions.go

```diff
@@ -2,6 +2,7 @@ package directives
 
 import (
 	argocd "github.com/akuity/kargo/internal/controller/argocd/api/v1alpha1"
+	"github.com/akuity/kargo/pkg/x/directives/builtins"
```

Comment:
We may want to take this opportunity to rename the package to singular (`.../directive/builtin`). As we (probably _I_, actually) appear to have missed the boat there to adhere to community conventions.

Reply:
Consider it done

Reply:
All set.